src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx

```diff
@@ -118,7 +125,8 @@ class ActionBarActionsButtonDrawerOption extends React.PureComponent {
                             id={option.id}
                             style={{
                                 backgroundColor: option.disabled ?
-                                    colorStyles.backgroundColorStatic :
+                                    // eslint-disable-next-line react/prop-types
```

Comment:
Why is this suppression necessary?
Do we have to add `grey` to the prop type declaration?
NAB but you know I have to ask

Reply:
Yeah, For supressing this typescript error
`'theme.palette.grey.400' is missing in props validation`